tests/testthat/test-app-function.R

```diff
@@ -0,0 +1,11 @@
+test_that("tfpbrowser app initial values are consistent", {
```

Comment:
This test fails for me locally

```r
Error in `$<-.data.frame`(`*tmp*`, "call_text", value = c("shinytest2::AppDriver$new(shiny_app, name = \"tfpbrowser\")",  : 
  replacement has 9 rows, data has 7
```

It also won't run in non-interactive setting, so the tests won't be usable in CI. Is this the intended setup?

Reply:
I've renamed the file to test-shinytest2.R to distinguish it from other {testthat} tests we may want to include (which should work side-by-side with shinytest2). 

Running `testthat::test_local()` locally doesn't give me that error, which is slightly odd. I get one warning saying that there are non-unique HTML values for treeview (this should be fixed in the convert-to-modules MR coming soon) One possible issue - shinytest2 requires that your package to be installed when testing, was the most recent version of {tfpbrowser} installed before running tests?

Removing the `interactive()` statement broke R CMD check locally (`'Shiny app did not become stable in 15000ms'`) so I'm okay with running it locally (for now)